crates/circuits/sha256-air/src/lib.rs

```diff
@@ -0,0 +1,4 @@
+//! Standalone AIR implementation of a single SHA256 block
+
+mod sha256;
+pub use sha256::*;
```

Comment:
I think we can follow https://github.com/Plonky3/Plonky3/tree/main/keccak-air/src and just not have this folder (move stuff to `src`)